src/App.js

```diff
@@ -59,14 +59,12 @@ export class App extends Component {
           imdbUrl: Website,
         };
 
-        this.setState(prevState => ({
-          filmsList: [...prevState.filmsList, newFilm],
-        }));
+        store.dispatch(findNewFilm(newFilm));
```

Comment:
Why do you need separate action for `findNewFilm`? Do you do something different inside reducer?